fili-core/src/test/groovy/com/yahoo/bard/webservice/web/endpoints/BaseDataServletComponentSpec.groovy

```diff
@@ -206,23 +203,22 @@ abstract class BaseDataServletComponentSpec extends Specification {
      */
     Response makeAbstractRequest(Closure queryParams=this.&getQueryParams) {
         // Set target of call
-        def httpCall = jtb.getHarness().target(getTarget())
+        def httpCall = jtb.harness.target(target)
```

Comment:
It would use the getter. For direct field access, you have to use the [field access operator](http://groovy-lang.org/operators.html#_direct_field_access_operator): 

```groovy
class User {
    public final String name                 
    User(String name) { this.name = name}
    String getName() { "Name: $name" }
}
def user = new User('Bob')
assert user.name == 'Name: Bob'   // Goes through the getter
assert user.@name == 'Bob' // Gets the field directly
```